cognoml/tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+import mock
+from cognoml import utils 
+
+@mock.patch('os.mkdir')
+@mock.patch('os.path.exists')
```

Comment:
I think I get it. The `mock_mkdir` parameter is related to `@mock.patch('os.mkdir')`. Then we check that when `os.mkdir` was called that `False` was returned. What happens if os.mkdir is called multiple times?